README.adoc

```diff
@@ -1,85 +1,90 @@
 
-
-image::images/IoT-OSE-Containers.png[width="462", height="614", align=center]
+image::images/IoT-OSE-Containers.png[height="400", align="center"]
 
 *Please note: This repository is currently under development and the feature set/components are subject to change without notice*
 
 == Components
 
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[JBOSS Fuse Integration Services]*
- A set of tools and container images that enable development, deployment, and management of microservices within OpenShift.
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[JBOSS A-MQ]*
-A-MQ message broker container image for communicating with Sensor using MQTT protocol
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-decision-server-image/chapter-1-introduction-to-the-decision-server-image/[JBOSS BRMS Decision Server]*
-Container image of the decision platform for executing realtime business rules
-- *https://zeppelin.apache.org/[Apache Zeppelin]*
-Container image of the tool for interactive data analytics.
-- *https://www.postgresql.org/[PostgreSQL]*
-Container image of the database for storing sensor data
-- *Simulated Software Sensor*
-Spring boot based software app simulating temperature, vibration and GPS sensor data
+The application is deployed as a set of microservices inside containers on OpenShift:
+
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[JBoss Fuse Integration Services]* (FIS): Tools that enable rapid development, deployment, and management of microservices within OpenShift.
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[JBoss A-MQ]*: Apache ActiveMQ-based message broker container image for receiving and storing telemetry from the Sensor via http://mqtt.org/[MQTT]
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-decision-server-image/chapter-1-introduction-to-the-decision-server-image/[JBoss BRMS Decision Server]*: Real-time execution of business rules for processing telemetry
+- *https://zeppelin.apache.org/[Apache Zeppelin]*: Interactive data visualization and analytics
+- *https://www.postgresql.org/[PostgreSQL]*: Provides persistence for sensor data
+- *Simulated Software Sensor*: A simple Spring Boot application that simulates temperature, vibration, and GPS sensor data
 
 == Prerequisites
 
-* Git
-* Access to an OpenShift environment
-* OpenShift Command Line Interface (CLI)
+* https://git-scm.com/[Git]
+* https://github.com/openshift/origin/releases[OpenShift command-line tools] for your platform
+* Access to an OpenShift environment.  Don't have one handy? Try https://asciinema.org/a/49402[`oc cluster up`] or the https://github.com/redhatdemocentral/cdk-install-demo[Red Hat Container Development Kit] (CDK)
```

Comment:
Preferably, yeah, but it's harder to keep up-to-date since the official docs are versioned - for example: https://access.redhat.com/documentation/en/openshift-container-platform/3.3/

Could also just drop the new stuff I added, since further down we mention the CDK anyway?